src/views/app/detail-panel/edit/parts/text-editor-container.tsx

```diff
@@ -52,7 +52,8 @@ export const TextEditorContainer: FC<TextEditorContainerProps> = ({
 
 	const composerCustomOptions = {
 		toolbar_sticky: true,
-		ui_mode: 'split'
+		ui_mode: 'split',
+		newline_behavior: 'linebreak'
```

Comment:
The usage of _newline_behaviour_ results in new issues:
- it seems impossible to create a new paragraph
- there is no way to create a bullet or a numbered list on a new line/paragraph
- pressing enter on a bullet or a numbered list doesn't create a new list item